### PR TITLE
Django module initial commit

### PR DESCRIPTION
**Pull request by Franco98404**
Could you review my code? Please

### CODE REVIEW

**Review by MatiasBonifacio — COMMENTED** [review]
Perfect, i think :D

**Review by ChristianTC — APPROVED** [review]
Good job!

**Comment by awielern** [comment]
It seems ok, but you have to check the url file.

**Review by MatiasBonifacio — APPROVED** [review]

File: CONVERT_SERVICE/convert_service/convert_service/urls.py [hunk 1]
```diff
@@ -13,9 +26,21 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from convert_service.views import index, upload, download
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_service/urls.py]: Please review PEP8 importing rules here.

Comment (MatiasBonifacio) [inline CONVERT_SERVICE/convert_service/convert_service/urls.py]: Pls use PEP 8 rules

**Review by MatiasBonifacio — APPROVED** [review]
It's okay only need to apply pep8 rules

**Review by lcarrietas — CHANGES_REQUESTED** [review]
Please check the comment :)

File: CONVERT_SERVICE/convert_service/convert_service/urls.py (hunk 1 above)
Comment (Franco98404) [reply CONVERT_SERVICE/convert_service/convert_service/urls.py]: Thank you, i will review it.

Comment (Franco98404) [reply CONVERT_SERVICE/convert_service/convert_service/urls.py]: Thank you, I will review the code.

**Review by lcarrietas — APPROVED** [review]
It seems good!

**Pull request merged**